core/extension/gdextension_manager.cpp

```diff
@@ -95,6 +105,17 @@ Ref<GDExtension> GDExtensionManager::get_extension(const String &p_path) {
 	return E->value;
 }
 
+bool GDExtensionManager::class_has_icon_path(const String &p_class) const {
```

Comment:
This is basically equivalent of `!class_get_icon_path().is_empty()`.

Reply:
Yeah, but it's explicit this way, so I don't mind.